named_entity_recognition/documentation/step-2-build-components.md

```diff
@@ -0,0 +1,75 @@
+# Build components
+
+A component is code that performs one step in the Kubeflow pipeline. It is a containerized implementation of an ML task. **Components can be reused in other pipelines.**
+
+## Component structure
+A component follows a specific structure and contains:
```

Comment:
Link to the docs where this structure is specified

Reply:
✓ done